cloudigrade/account/reports/helper.py

```diff
@@ -0,0 +1,115 @@
+"""Cloud-specific helpers to aid in cloud-agnostic report processing."""
+from abc import ABC, abstractmethod
+
+from django.db import models
+from django.utils.translation import gettext as _
+
+from account import CLOUD_PROVIDERS
+from account.exceptions import InvalidCloudProviderError
+from account.models import AwsAccount
+
+
+def get_report_helper(cloud_provider, cloud_account_id):
+    """
+    Get a cloud-specific ReportHelper instance.
+
+    Args:
+        cloud_provider (str): The cloud provider
+        cloud_account_id (str): The cloud-specific account ID
+
+    Returns:
+        ReportHelper: An instance of a cloud-specific ReportHelper child class.
+
+    """
+    if cloud_provider not in CLOUD_PROVIDERS:
+        raise InvalidCloudProviderError(
+            _('Unsupported cloud provider "{0}".').format(cloud_provider)
```

Comment:
are we not doing the `f'{cloud_provider}'` string format method anymore?

Reply:
Yes, I mentioned this in a previous PR. I believe f-strings often don't work for gettext because of the order of execution. Which reminds me I need to file an issue to go fix/clean up all our incorrect f-strings…

Take this for example:
```
astronaut = 'Dave'
message = _(f"I'm sorry, {astronaut}.")
```
The value passed into the `_` function would have already been substituted by the Python interpreter, meaning the translation function gets `"I'm sorry, Dave."` which should not be found in the translation mapping.